crates/stackable-operator/src/commons/affinity.rs

```diff
@@ -51,6 +60,10 @@ pub struct StackableNodeSelector {
 
 impl Atomic for StackableNodeSelector {}
 
+pub fn stackable_node_selector_schema(gen: &mut schemars::gen::SchemaGenerator) -> Schema {
```

Comment:
note: Can we maybe add a small comment to this function to better understand where and how this is used?

Reply:
I tried containing all the docs in a single place on `StackableNodeSelector`.
WDYT of

We need a custom JsonSchema for StackableNodeSelector, please have a look at the documentation there

Reply:
Yeah something along those lines is fine!

Reply:
Added, please have a look